plugin/httptrace/server_test.go

```diff
@@ -0,0 +1,57 @@
+package httptrace_test
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"go.opentelemetry.io/exporter/trace/jaeger"
+	"go.opentelemetry.io/plugin/httptrace"
+	"go.opentelemetry.io/sdk/trace"
+)
+
+func ExampleNewHandler() {
+	trace.Register()
+
+	// Create Jaeger exporter to be able to retrieve
+	// the collected spans.
+	exporter, err := jaeger.NewExporter(jaeger.Options{
```

Comment:
Should this be stdout exporter instead of Jaeger to be consistent with other example?

Reply:
changing.